src/display/editor/ink.js

```diff
@@ -484,6 +495,10 @@ class InkEditor extends AnnotationEditor {
   #createCanvas() {
     this.canvas = document.createElement("canvas");
     this.canvas.className = "inkEditorCanvas";
+
+    InkEditor._l10nPromise
+      .get("editor_ink_canvas_aria_label")
+      .then(msg => this.canvas.setAttribute("aria-label", msg));
```

Comment:
Given the asynchronicity of the code, would it be safer to do the following?
```suggestion
      .then(msg => this.canvas?.setAttribute("aria-label", msg));
```